### ISSUE TITLE
it doesn't work on wordpress 5.6.2

### ISSUE THREAD

**Issue by avnkhanh** [iranimij/automatic-upload-images#5]
After updating Wordpress version 5.6.2, images are not saved to the media library

**Comment by iranimij** [comment]
hi Avnkhanh 
Our plugin is compatible with WordPress version 5.6.2, 
what's your problem with uploading images? Do you get any errors?
Is it possible to send me the page that you want to upload it's images ?

**Comment by avnkhanh** [comment]
ok my friend
this is link
https://phuongnam.info/wiki/kien-thuc/mtu-va-mss-nhung-dieu-ban-can-biet/

Wordpress 5.6.2
Multisite
HelpGuru theme
Post at  Knowledge Based ( it same Article )

if you need Admin user to check, talk w me agian.

Vào CN, 28 thg 2, 2021 vào lúc 11:29 Iman Heydari <
notifications@github.com> đã viết:

**Comment by iranimij** [comment]
I tested this link, But it was ok and I could upload images,
https://prnt.sc/10ay9vu
Would you please send me your admin data for checking this problem on your site?

**Comment by iranimij** [comment]
Did you install Classic Editor plugin on your site? It seems there is a conflict between these two plugins, I have to check it.

**Comment by avnkhanh** [comment]
Yes, this plguin : Advanced Editor Tools (previously TinyMCE Advanced)
and orther Article is ok 👍 
https://phuongnam.info/wiki/kien-thuc/che-do-bridge-mode-la-gi-va-tai-sao-toi-can-no/
https://phuongnam.info/wiki/wp-content/uploads/sites/15/2021/03/what_is_bridge_mode.png

Thanks

**Comment by iranimij** [comment]
@avnkhanh We have an update please check again.
